applications/TrilinosApplication/tests/test_trilinos_redistance.py

```diff
@@ -83,7 +80,7 @@ def testTrilinosRedistance(self):
         trilinos_linear_solver = trilinos_linear_solver_factory.ConstructSolver(
             KratosMultiphysics.Parameters("""{"solver_type" : "amesos" }"""))
 
-        epetra_comm = TrilinosApplication.CreateCommunicator()
+        epetra_comm = TrilinosApplication.CreateEpetraCommunicator(KratosMultiphysics.DataCommunicator.GetDefault())

```

Comment:
Note that I am only using this one FORE TESTING!
It should not be used in production code!